examples/misc/lib/language_tour/control_flow.dart

```diff
@@ -166,7 +168,7 @@ void miscDeclAnalyzedButNotTested() {
   {
     int number = 0;
     String urlString = '';
-    String? text;
+    String? text = nullableValue<String>();
```

Comment:
I'm curious why the `nullsbleValue` instead of just using a null?

Reply:
This is to trick the analyzer so it doesn't think the assert below is unnecessary.